src/artemis/fast_grid_scan_plan.py

```diff
@@ -62,11 +62,11 @@ def run_gridscan(
     yield from update_params_from_epics_devices(
         parameters, undulator, synchrotron, slit_gap
     )
-    config = "config"
+    ispyb_config = os.environ["ISPYB_CONFIG_PATH"]
```

Comment:
**should**: We can just add the default here rather than in the test itself using `print(os.environ.get('KEY_THAT_MIGHT_EXIST', default_value))`